app/components/primer/icon_button.rb

```diff
@@ -8,38 +8,22 @@ module Primer
   #   The `aria-label` should describe the action to be invoked rather than the icon itself. For instance,
   #   if your `IconButton` renders a magnifying glass icon and invokes a search action, the `aria-label` should be
   #   `"Search"` instead of `"Magnifying glass"`.
+  #   Internally the `aria-label` will be used for the `Tooltip` text. The `button` element is associated via `aria-labelledby`
+  #   to the `Tooltip` so it will serve as the `aria-label` for the `button`.
```

Comment:
We should also add support for tooltip with `type: :description` to support cases like the icon buttons we have in the markdown toolbar. We will probably want to add a note/example of that too.

I'm not sure what a good API for this. Maybe we will need the tooltip slot for that?

Reply:
Added support for `aria-description`, let me know what you think!